apps/chameleon/app/api/send/route.ts

```diff
@@ -0,0 +1,33 @@
+import { Resend } from 'resend'
+
+import { EmailTemplate } from '@/components/email-template'
+
+const resend = new Resend(process.env.RESEND_API_KEY)
```

Comment:
lets add this to file apps/chameleon/env.mjs then import it like the rest of the envs